src/main/java/core/basesyntax/ApplicationRun.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.dto.FruitRecordDto;
+import service.FruitOperationsService;
+import service.ParserService;
+import service.ReaderService;
+import service.ReportService;
+import service.impl.AddOperation;
+import service.impl.ParserServiceImpl;
+import service.impl.ReaderServiceImpl;
+import service.impl.RemoveOperation;
+import service.impl.ReportServiceImpl;
+
+public class ApplicationRun {
+    private static final String SUPPLY = "s";
+    private static final String PURCHASE = "p";
+    private static final String RETURN = "r";
+    private static final String PATH_SOURCE_FILE = "src/main/resources/input.csv";
+    private static final String OUTPUT_FILE_NAME = "src/main/resources/output.csv";
+    private static final Map<String, FruitOperationsService> fruitOperationsServiceMap
+            = new HashMap<>();
+
+    static {
+        fruitOperationsServiceMap.put(SUPPLY, new AddOperation());
```

Comment:
You create new AddOperation() two times and will do one more time for balance just create one object and pass where needed